src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -276,3 +276,33 @@
         <BitChoiceOption Text="Disabled option" Value="3" IsEnabled="false"></BitChoiceOption>
     </BitChoiceGroup>
 </div>
+<hr />
+
+<div class="toggle-container">
+ <h2>BitToggle</h2>
+ <h3>Basics</h3>
+ <BitToggle @bind-IsChecked="IsToggleChecked" IsEnabled="true" OnText="On" OffText="Off">Enabled and checked</BitToggle>
```

Comment:
did you consider the 2-way bound parameter (IsChecked) in all of these samples?

Reply:
Not. I do this